src/main/webapp/TempMarker.js

```diff
@@ -61,7 +61,7 @@ class TempMarker {
    * @Private
    * Removes the temp marker from the map.
    */
-  remove_() {
+  remove() {
```

Comment:
Do you need to change the visibility of this method?

Reply:
Oops, I should have caught that, my mistake.